langchain/src/document_loaders/tests/csv-blob.test.ts

```diff
@@ -1,6 +1,6 @@
-import url from "node:url";
-import path from "node:path";
-import fs from "node:fs/promises";
+import * as url from "node:url";
```

Comment:
this is great

prettier is super powerful w/ imports btw; I recommend using one of the prettier import sorting plugins ala https://github.com/transitive-bullshit/chatgpt-api/blob/main/.prettierrc.cjs

just completely removes having to think about imports and ordering best practices and is a huge time saver

Reply:
Oh need to set that up